.ci/travis_build_opencv.sh

```diff
@@ -1,20 +1,15 @@
 #!/bin/bash
 set -eux -o pipefail
 
-OPENCV_VERSION=${OPENCV_VERSION:-3.4.0}
-URL=https://github.com/opencv/opencv/archive/$OPENCV_VERSION.zip
-URL_CONTRIB=https://github.com/opencv/opencv_contrib/archive/$OPENCV_VERSION.zip
-OPENCV_BUILD=$(pwd)/opencv-$OPENCV_VERSION/build
-OPENCV_CONTRIB=$(pwd)/opencv_contrib-$OPENCV_VERSION/modules
+OPENCV_VERSION=${OPENCV_VERSION:-3.4.1}
+OPENCV_BUILD=$(pwd)/opencv/build
```

Comment:
I was keeping version to make sure that caching is working with proper version. That's probably is not required, but worth to be mentioned.